bionic/utils/misc.py

```diff
@@ -365,6 +365,21 @@ def __init__(self):
         self.values = set()
         self.lock = threading.Lock()
 
+    def __getstate__(self):
```

Comment:
Does this actually work the way we'd expect, or is this just a hack to avoid throwing an exception?